pyart/retrieve/echo_class.py

```diff
@@ -106,6 +105,213 @@ def steiner_conv_strat(grid, dx=None, dy=None, intense=42.0,
                           '2 = Convective')}
 
 
+def conv_strat(grid, dx=None, dy=None, level_m=None, always_core_thres=42, bkg_rad_km=11,
```

Comment:
Can you name this `conv_strat_yuter` or `conv_strat_yuter_houze`? This would make it more clear to the user which algorithm is being applied.